bin/nginx.conf

```diff
@@ -10,11 +10,19 @@ server {
 
 # now we declare our main server
 server {
-
     listen 80;
-    server_name effectivethesis.com beta.effectivethesis.com localhost;
+    listen 443 ssl;
+    server_name beta.effectivethesis.com localhost;
+
+    ssl_certificate /etc/letsencrypt/live/beta.effectivethesis.com/fullchain.pem;
```

Comment:
tady jsem chtěl použít `$host`, ale v tu chvíli nginx házel permission denied (i přesto, že to vedlo na stejný file) :disappointed: . 

Takže asi budeme muset mít dva různé servery - jeden pro produkci a druhý pro beta.

Reply:
Druha moznost je tam tu cestu davat dynamicky pri deployi.